core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -740,6 +740,30 @@ public MetricKey build() {
           .setMetricType(MetricType.GAUGE)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey WORKER_BLOCK_REMOVER_BLOCKS_TO_REMOVED_TAKE_COUNT =
+      new Builder(Name.WORKER_BLOCK_REMOVER_BLOCKS_TO_REMOVED_TAKE_COUNT)
+          .setDescription("Total number of asyncBlockRemover take block count")
```

Comment:
The total number of blocks tried to be removed from this worker by asynchronous block remover.

Reply:
Similar for all other metric key names and descriptions